server/controllers/refer/peopleController.ts

```diff
@@ -0,0 +1,29 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
+import createError from 'http-errors'
+
+import type PersonService from '../../services/personService'
+import { typeUtils } from '../../utils'
+import personUtils from '../../utils/personUtils'
+
+export default class PeopleController {
+  constructor(private readonly personService: PersonService) {}
+
+  show(): TypedRequestHandler<Request, Response> {
+    return async (req: Request, res: Response) => {
+      typeUtils.assertHasUser(req)
+
+      const person = await this.personService.getPerson(req.user.token, req.params.prisonNumber)
+
+      if (!person) {
+        throw createError(404, {
+          userMessage: `Person with prison number ${req.params.prisonNumber} not found.`,
+        })
+      }
```

Comment:
I wonder if we might need to prettify this later. If this is the page users reach from a search, a 404 might not be ideal (versus a "No results" page). But that's probably one for later!

Reply:
Very good point. I'll add a card to address this in a future PR as this one is large enough.